tests/ci_tests/code_formatter/conftest.py

```diff
@@ -0,0 +1,28 @@
+"""Fixtures and configuration for code_formatter tests"""
+
+import os
+from unittest import mock
+
+import pytest
+
+# Set required environment variables before modules are imported
+os.environ.setdefault("GOOGLE_API_KEY", "fake_test_key")
+os.environ.setdefault("LANGFUSE_SECRET_KEY", "fake_secret")
+os.environ.setdefault("LANGFUSE_PUBLIC_KEY", "fake_public")
+os.environ.setdefault("LANGFUSE_HOST", "http://localhost")
+os.environ.setdefault("GITHUB_TOKEN", "fake_github_token")
+# Disable OpenTelemetry tracing during tests
+os.environ.setdefault("OTEL_SDK_DISABLED", "true")
+
+
+@pytest.fixture
```

Comment:
Added type hinting to the mock_langfuse_client function as per style guide's direction to improve readability and detect errors early.
```suggestion
def mock_langfuse_client() -> mock.MagicMock:
        """Provide a mock Langfuse client."""
```